kernel/test/unit/engine.test.tsx

```diff
@@ -0,0 +1,28 @@
+import { expect } from 'chai'
+import { Entity, BoxShape, Transform, Vector3, engine } from 'decentraland-ecs/src';
+
+describe.skip('Engine tests', () => {
```

Comment:
```suggestion
describe('Engine tests', () => {
```

Reply:
@pravusjif the test was skipped by code. please pull and run it in the UI to verify

Reply:
It failed in the CI :( 
```
  1) Engine tests
       getComponentGroupReturnsCachedGroup:
     TypeError: c.expect(...).to.same is not a function
      at Context.<anonymous> (test/out/index.js:661:3011493)
```